src/com/xilinx/rapidwright/edif/EDIFNetlist.java

```diff
@@ -1610,9 +1612,19 @@ public void expandMacroUnisims(Series series) {
                         if (!isHDILib) {
                             Pair<String, EnumSet<IOStandard>> exception = seriesMacroExpandExceptionMap.get(cellName);
                             if (exception != null) {
-                                EDIFPropertyValue value = getIOStandard(inst);
-                                IOStandard ioStandard = IOStandard.valueOf(value.getValue());
-                                if (exception.getSecond().contains(ioStandard)) {
+                                Boolean expand = null;
+                                for (EDIFPropertyValue value : getIOStandards(inst)) {
+                                    IOStandard ioStandard = IOStandard.valueOf(value.getValue());
+                                    boolean contained = exception.getSecond().contains(ioStandard);
+                                    if (expand == null) {
+                                        expand = contained;
+                                    } else {
+                                        if (expand != contained) {
+                                            throw new RuntimeException();
```

Comment:
Perhaps we could add a message to this exception?

Reply:
Ooh yes, forgot about that one!